src/firestore/fsConfig.ts

```diff
@@ -0,0 +1,78 @@
+import { FirebaseError } from "../error";
+import { logger } from "../logger";
+import { Options } from "../options";
+
+export interface ParsedFirestoreConfig {
+  database: string;
+  rules?: string;
+  indexes?: string;
+}
+
+export function getFirestoreConfig(projectId: string, options: Options): ParsedFirestoreConfig[] {
+  const fsConfig = options.config.src.firestore;
+  if (fsConfig === undefined) {
```

Comment:
Why `=== undefined` instead of just a falsy check?

Reply:
main motivation was to stay consistent with rtdb. Either would work however there is an edge case if fsConfig is an empty object `{}` we will miss adding an empty configuration for (default). there should not be any repercussions to this (either way deploy would be a no-op) but my preference would be to stay as close to the rtdb implementation as possible